yt/frontends/boxlib/data_structures.py

```diff
@@ -922,6 +922,53 @@ def relative_refinement(self, l0, l1):
         return self.refine_by ** (l1 - l0 + offset)
 
 
+class AMReXHierarchy(BoxlibHierarchy):
+    def __init__(self, ds, dataset_type="boxlib_native"):
+        super().__init__(ds, dataset_type)
+
+        if "particles" in self.ds.parameters:
+            is_checkpoint = True
+            for ptype in self.ds.particle_types:
+                self._read_particles(ptype, is_checkpoint)
+
+
+class AMReXDataset(BoxlibDataset):
+    _index_class: Type[BoxlibHierarchy] = AMReXHierarchy
+    _subtype_keyword = "amrex"
+    _default_cparam_filename = "job_info"
+
+    def __init__(
```

Comment:
I have doubts that this override is any use, as it does nothing besides calling the overridden function. Am I missing something ?

Reply:
I'm pretty sure that's correct, as all the default parameters are the same. I'll remove it.